RecoLocalCalo/HcalRecProducers/python/HBHEMethod3Parameters_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+# Configuration parameters for Method 3
+pedestalSubtractionType = cms.int32(1)
```

Comment:
you can wrap this in a PSet and then include it in the module (has to be included before any named parameters of the module
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Parameter_Set_PSet_Objects
)
https://github.com/cms-sw/cmssw/compare/cms-sw:CMSSW_7_6_X...slava77:CMSSW_7_6_0_pre7/sign613/alt-11769